app/services/Athena.scala

```diff
@@ -203,7 +203,7 @@ class Athena() extends StrictLogging {
           .flatMap(row => {
             val data = ArticleEpicData(row)
             if (data.isEmpty) {
-              logger.error(s"Failed to parse row from athena: $row")
+              logger.info(s"Failed to parse row from athena: $row")
```

Comment:
why log at all if it's an expected outcome?

Reply:
Good point.
And actually I just tested this branch in CODE and the improved logging suggests it's because _all_ queries return a row with the column names!
`Map(date_hour -> date_hour, variant -> variant, views -> views)`
So I'm going to find a smarter solution